site/docs/api-quickstart.md

```diff
@@ -102,13 +102,13 @@ When using the Iceberg API directly, type IDs are required. Conversions from oth
 
 When a table is created, all IDs in the schema are re-assigned to ensure uniqueness.
 
-### Convert a schema from Avro
+### Convert a schema from Apache Avro
 
-To create an Iceberg schema from an existing Avro schema, use converters in `AvroSchemaUtil`:
+To create an Iceberg schema from an existing [Apache Avro schema](https://avro.apache.org/), use converters in `AvroSchemaUtil`:
```

Comment:
No need to link here if it isn't to a specific documentation page about the schema. It's easy enough to find Avro and this documentation is targeted at people that already know.

Reply:
Reverted the change.